CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 None
+<a name="3.0.3"</a>
+## [3.0.1] 2021-11-4
```

Comment:
version number here should be 3.0.3

Reply:
Also - need to update the file that actually tracks the version. Changelog is purely for documentation purposes.